transposon/analyzeBCcounts.sh

```diff
@@ -301,6 +301,18 @@ for i in `echo {10000,50000,100000,250000,500000,1000000,2000000,3000000,4000000
     echo "$i $saturation minreads${minReadCutoff}" 
 done > ${OUTDIR}/${sample}.Saturation.minReads_${minReadCutoff}.txt
 
+if [ "${minCellBCLength}" -gt 0 ]; then
```

Comment:
This generates a new output file for 10xRNA libraries -- can you comment on its format and purpose?

Will Flowcell_Info.sh need to be updated too?

Reply:
Sure,

The format is the same as the previous saturation curve, three column file:
- number of reads
- number of BC+cellBC pairs
- read cutoff

Regarding Flowcell_Info.sh, upon review it will probably will need to be review if we keep this strategy. Instead we could fully replace the current saturation file for the new one. Would rather do that? Or do you want me to explore some review options to Flowcell_Info.sh?

Reply:
I don't think we need saturation curves for both BC and BC+cellBC, the latter is probably fine

Reply:
Okay

Reply:
I've consolidated the saturation curve computation on the latest commit. It ran okay on my test